packages/polyfills-loader/src/injectPolyfillsLoader.ts

```diff
@@ -96,8 +99,8 @@ export async function injectPolyfillsLoader(
 ): Promise<InjectPolyfillsLoaderResult> {
   const documentAst = parse(htmlString);
 
-  const headAst = findElement(documentAst, e => getTagName(e) === 'head') as unknown as Node;
-  const bodyAst = findElement(documentAst, e => getTagName(e) === 'body') as unknown as Node;
+  const headAst = findElement(documentAst, e => getTagName(e) === 'head') as Element;
+  const bodyAst = findElement(documentAst, e => getTagName(e) === 'body') as Element;
```

Comment:
you may not need this cast anymore, it should be inferred as `Element|null` and your `if` would handle the null below

Reply:
Removed the type casts.